server/protocols/jmap/src/main/java/org/apache/james/jmap/UriPathTemplate.java

```diff
@@ -151,7 +152,7 @@ private Matcher matcher(String uri) {
             return pathParameters;
         }
 
-        pathParameters = new HashMap<>();
+        pathParameters = new ConcurrentHashMap<>();
```

Comment:
is it mutated somewhere?
I would rather create an ImmutableMap if not

Reply:
Let's not modify this copy/pasted code?

Reply:
it was not a `ConcurrentHashMap` before, right?

Reply:
Yes I did change to remi's remark before. I just dropped that fixup. Can we agree on the fact that we do not want to modify this copy/pasted code for now? :))